geo/Cargo.toml

```diff
@@ -32,11 +33,12 @@ proj-network = ["use-proj", "proj/network"]
 use-serde = ["serde", "geo-types/serde"]
 
 [dev-dependencies]
+pretty_env_logger = "*"
```

Comment:
It's helpful to tune the logging when running tests. This is my go-to, but I could be convinced otherwise.

Reply:
sgtm. can we lock it to a version so our tests don't break if there's ever breaking changes?

Reply:
done!